Content.Server/_Goobstation/Changeling/ChangelingSystem.Abilities.cs

```diff
@@ -717,6 +719,12 @@ public void OnLesserForm(EntityUid uid, ChangelingComponent comp, ref ActionLess
 
         var targetUid = (EntityUid)newUid;
 
+        var uiComp = EnsureComp<UserInterfaceComponent>(targetUid);
+        if (!_userInterfaceSystem.HasUi(targetUid, StoreUiKey.Key, uiComp))
+        {
+            _userInterfaceSystem.SetUi(targetUid, StoreUiKey.Key, new InterfaceData("StoreBoundUserInterface", 2, true));
```

Comment:
for the `InterfaceData`, 2 and true are default values so there's no need to pass those - this goes for the other files too